test/plugin_functional/plugins/saved_object_export_transforms/server/plugin.ts

```diff
@@ -31,7 +31,7 @@ export class SavedObjectExportTransformsPlugin implements Plugin {
       management: {
         defaultSearchField: 'title',
         importableAndExportable: true,
-        getTitle: (obj) => obj.attributes.title,
+        getTitle: (obj) => (obj.attributes as any).title,
```

Comment:
Why are you forced to cast to `any` given you specify the generic type L19?

Reply:
yeah I should have removed that change, was left-over from debugging.

> I just wonder how many more files will need to be updated before typecheck stop complaining for errors similar to

Strangely enough `node scripts/type_check` passes locally. More generally I'm just trying to fix the low hanging fruit, for instance `dashboard` has tons of `SavedObjectAttributes` and don't seem to have a dedicated type that describes their attributes so I left that plugin alone.